vapi/libecal-1.2.vapi

```diff
@@ -141,7 +141,7 @@ namespace ECal {
 		public void get_description_list (out GLib.SList<ECal.ComponentText> text_list);
 		public void get_dtend (out ECal.ComponentDateTime? dt);
 		public void get_dtstamp (out ICal.Time? t);
-		public void get_dtstart (out ECal.ComponentDateTime? dt);
+		public void get_dtstart (out ECal.ComponentDateTime dt);
```

Comment:
This change does not appear to be essential for the fix.

Reply:
This solves the mentioned VAPI problem in the description of the requests. It seems unecessary for resolving this issue since this request also removes "update_event" call and uses "remove_event" call. I can discard this change for this pull request but a VAPI change may be required in the future.